src/network/networkd-manager.c

```diff
@@ -128,9 +128,7 @@ static int match_prepare_for_sleep(sd_bus_message *message, void *userdata, sd_b
 
         log_debug("Coming back from suspend, resetting all connections...");
 
-        manager_reset_all(m);
-
-        return 0;
+        return manager_reset_all(m);
```

Comment:
Hmm, this is used as a callback. I think ignoring the errors is the right thing to do here. sd_bus_add_match(3) says that 0 means "continue calling other callbacks", and I think that's what we should keep doing.

Reply:
OK. Thank you.